lib/commands/devices/index.ts

```diff
@@ -99,31 +100,52 @@ export default class DevicesCmd extends Command {
 			return device;
 		});
 
-		const fields = [
-			'id',
-			'uuid',
-			'device_name',
-			'device_type',
-			'fleet',
-			'status',
-			'is_online',
-			'supervisor_version',
-			'os_version',
-			'dashboard_url',
-		];
-
-		if (options.json) {
-			const { pickAndRename } = await import('../../utils/helpers');
-			const mapped = devices.map((device) => pickAndRename(device, fields));
-			console.log(JSON.stringify(mapped, null, 4));
+		if (isV14()) {
+			const outputFields = [
```

Comment:
Ok.  My strategy was to modify the old code as little as possible, even if this led to code duplication (since any duplication would only be temporary).